runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -1175,6 +1184,11 @@ class CompilationInfo
    uint8_t _chTableUpdateFlags;
    // number of local gc cycles done
    uint32_t _localGCCounter = 0;
+#if defined(JITSERVER_SUPPORT)
+   std::string                   _sslRootCerts;
```

Comment:
Are these entities used only by the server? If so we could move them in CompilationInfoPerThreadRemote.

Reply:
_sslRootCerts is used by the client; 
_sslKeys and _sslCerts used by the server;